TDD_Training/UnitTestProject/FindMultiples.Tests/FindMultiplesTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+using Xunit;
+using System.Threading.Tasks;
+
+namespace UnitTestProject.FindMultiples.Tests
+{
+    /// <summary>
+    /// Summary description for FindMultiplesTests
+    /// </summary>
+    public class FindMultiplesTests
```

Comment:
I couldn't see the need for these test cases?
Can you explain??

Reply:
This is the exact folder where I wrote test cases. The above one (I mean the first PR comment) that file is not being used. I removed it actually. In my local, I don't find that above file. I will check one more time and push the changes.